src/Jaeger.Core/Configuration.cs

```diff
@@ -57,6 +57,26 @@ public class Configuration
         /// </summary>
         public const string JaegerAgentPort = JaegerPrefix + "AGENT_PORT";
 
+        /// <summary>
+        /// The target used to locate the grpc collector endpoint.
+        /// </summary>
+        public const string JaegerGrpcTarget = JaegerPrefix + "GRPC_TARGET";
+
+        /// <summary>
+        /// The root certificate file used to check the server side certificate from GRPC collector (roots.pem).
+        /// </summary>
+        public const string JaegerGrpcRootCertificate = JaegerPrefix + "GRPC_ROOT_CERTIFICATE";
```

Comment:
Q: is it, strictly speaking, scoped to gRPC only? Wouldn't it be possible to use the same certificates with HTTPS?

Reply:
You are technically right. But right now, it's only used there. Do we have environment variables for HTTPS in other clients? If yes, I would try to keep it in sync.

Reply:
I don't believe mTLS is supported in any other clients.

Reply:
Then I will for now just merge it. For 1.0.0 we might look into it again.